com.meetfranz.Franz.json

```diff
@@ -0,0 +1,57 @@
+{
+    "app-id": "com.meetfranz.Franz",
+    "base": "org.electronjs.Electron2.BaseApp",
+    "base-version": "18.08",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "franz",
+    "separate-locales": false,
+    "rename-desktop-file": "franz.desktop",
+    "rename-icon": "franz",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=dri",
+        "--filesystem=home",
+        "--talk-name=org.freedesktop.Notifications"
+    ],
+    "modules": [
+        {
+            "name": "franz-desktop",
+            "buildsystem": "simple",
+            "build-commands": [
+                "ar x franz_*.deb",
+                "rm -f franz_*.deb",
+                "tar xf data.tar.xz",
+                "rm -f control.tar.gz data.tar.xz debian-binary",
+                "cp -r usr/* opt/* /app",
+                "chmod -R a-s,go+rX,go-w /app/Franz",
+                "sed -i 's|Exec=.*|Exec=franz %U|' /app/share/applications/franz.desktop",
+                "install franz.sh /app/bin/franz",
+                "install -Dm644 com.meetfranz.Franz.appdata.xml /app/share/appdata/com.meetfranz.Franz.appdata.xml"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "only-arches": ["x86_64"],
+                    "url": "https://github.com/meetfranz/franz/releases/download/v5.0.0-beta.24/franz_5.0.0-beta.24_amd64.deb",
```

Comment:
Flathub is about stable and reproducible builds. I don't have any idea if this beta version is considered as stable?

Reply:
good question. I'm using it, but it is clearly marked as beta. I've sent a note to the author of the code.